dockerfiles/Dockerfile

```diff
@@ -1,17 +1,20 @@
-FROM centos:8
+FROM php:7.4-fpm
+
+ENV PHP_EXTRA_CONFIGURE_ARGS \
+  --enable-fpm \
+  --with-fpm-user=www-data \
+  --with-fpm-group=www-data \
+  --enable-intl \
+  --enable-opcache \
+  --enable-zip \
+  --enable-calendar
 
 # Install some must-haves
-RUN yum -y install vim wget sendmail
-RUN yum -y install git
+RUN apt-get install vim wget sendmail
```

Comment:
Docker fails at this stage with the message
```
Step 3/21 : RUN apt-get install vim wget sendmail
 ---> Running in 7250fefe7bd2
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package vim
E: Unable to locate package wget
E: Unable to locate package sendmail
```

Reply:
Sorry, I will fix it